dissect/target/plugins/os/windows/tasks.py

```diff
@@ -507,3 +1010,97 @@ def parse_task(self, entry):
+        at_task_data.idle_wait = minutes_duration_to_iso(at_task_data.idle_wait)
+        at_task_data.idle_deadline = minutes_duration_to_iso(at_task_data.idle_deadline)
+        at_task_data.task_flags.stop_on_idle_end = True if at_task_data.task_flags.stop_on_idle_end else False
+        at_task_data.task_flags.restart_on_idle = True if at_task_data.task_flags.restart_on_idle else False
+        at_task_data.max_run_time = round(at_task_data.max_run_time / 60000)
+        at_task_data.max_run_time = minutes_duration_to_iso(at_task_data.max_run_time)
+        at_task_data.task_flags.only_idle = True if at_task_data.task_flags.only_idle else False
+
+        # check which prio bit is set
+        for key, value in at_task_data.task_prio._values.items():
+            if value:
+                task_priority = key
+
+        record = TaskRecord(
+            uri=None,
+            security_descriptor=None,
+            source=entry,
+            date=None,
+            author=at_task_data.author,
+            version=at_task_data.file_version,
+            description=at_task_data.comment,
+            documentation=None,
+            principal_id=None,
+            user_id=None,
+            logon_type=None,
+            display_name=None,
+            run_level=None,
+            process_token_sid_type=None,
+            required_privileges=None,
+            allow_start_on_demand=None,
+            restart_on_failure_interval=at_task_data.retry_interval,
+            restart_on_failure_count=at_task_data.retry_count,
+            mutiple_instances_policy=None,
+            dissalow_start_on_batteries=at_task_data.task_flags.disallow_battery,
+            stop_going_on_batteries=at_task_data.task_flags.stop_battery,
+            start_when_available=None,
+            network_profile_name=None,
+            run_only_network_available=at_task_data.task_flags.internet_connected,
+            wake_to_run=at_task_data.task_flags.wake_to_run,
+            enabled=task_enabled,
+            hidden=at_task_data.task_flags.hidden,
+            delete_expired_task_after=None,
+            idle_duration=at_task_data.idle_wait,
+            idle_wait_timeout=at_task_data.idle_deadline,
+            idle_stop_on_idle_end=at_task_data.task_flags.stop_on_idle_end,
+            idle_restart_on_idle=at_task_data.task_flags.restart_on_idle,
+            network_settings_name=None,
+            network_settings_id=None,
+            execution_time_limit=at_task_data.max_run_time,
+            priority=task_priority,
+            run_only_idle=at_task_data.task_flags.only_idle,
+            unified_scheduling_engine=None,
+            disallow_start_on_remote_app_session=None,
+            data=at_task_data.user_data,
+            _target=self.target,
+        )
+
+        yield record
+
+        # Actions
+        for entry in at_task.get_at_actions():
+            grouped = GroupedRecord("filesystem/windows/task/grouped", [record, entry])
```

Comment:
```suggestion
        for task in at_task.get_at_actions():
            grouped = GroupedRecord("filesystem/windows/task/grouped", [record, task])
```
as `entry` has a specific meaning in this function.